roles/openshift_hosted/tasks/create_projects.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Create default projects
```

Comment:
This kind of feels out of place.  This should only really be creating the projects for the openshift_hosted router/registry.  Having it take `openshift_default_projects` and `openshift_additional_projects` seems a bit confusing.

Each role that requires a project (e.g. logging, metrics, hosted (default project)) should be in charge of creating their own projects.  That way any time they are run they can be consolidated into their own role and not have to call `create_projects.yml` from this role.

I'm fine with this for now but I'd be surprised if what I described above wasn't already the case.

Reply:
The issue we have is the 'openshift_additional_projects'

Those need to be created somewhere.  We can't just remove that functionality altogether.